src/datreant/core/limbs.py

```diff
@@ -500,6 +499,8 @@ def add(self, categorydict=None, **categories):
             outcats.update(categorydict)
         elif categorydict is None:
             pass
+        elif isinstance(categorydict, Categories):
```

Comment:
Instead of this, we could just modify line 498 to be `if isinstance(categorydict, (dict, Categories)):`, since one can just do `outcats.update(categorydict)` and this will work with a `Categories` object.